README.md

```diff
@@ -214,13 +221,14 @@ Managing UIDs between a docker container and a host volume can be complicated.
 
 ## Ports
 
-| Port  | Service      | Description          |
-|-------|--------------|----------------------|
-| 5432  | postgresql   | database access port |
-| 8000  | horizon      | main http port       |
-| 6060  | horizon      | admin port           |
-| 11625 | stellar-core | peer node port       |
-| 11626 | stellar-core | main http port       |
+| Port  | Service      | Description               |
+|-------|--------------|---------------------------|
+| 5432  | postgresql        | database access port |
+| 8000  | horizon           | main http port       |
+| 8004  | soroban rpc admin | main http port       |
+| 6060  | horizon           | admin port           |
```

Comment:
We should label the port as the admin port instead of as the main port.

Could we also number the ports so there's consistency between what we use for main vs admin ports for horizon and soroban-rpc on quickstart?

We can update the list of services that are available on the main http port too while we're at it.

For example, suggest:
```suggestion
| 8000  | horizon, soroban-rpc, friendbot | main http port       |
| 6060  | horizon                         | admin port           |
| 6061  | soroban-rpc                     | admin port       |
```

Reply:
yes, good suggestion, done - https://github.com/stellar/quickstart/pull/463/commits/cccbf5c5abcee280fc2d88a69959799851aa5d2c